modules-disabled/architecture/pages/pubcloud-requirements.adoc

```diff
@@ -1,6 +1,8 @@
 [[public-cloud-requirements]]
 = Public Cloud Requirements
 
+// Deprecated. See Quick Start: Public Cloud Guide instead. --LKB 2021-01-13
```

Comment:
Please do not add comments, since they break translations in some cases

Reply:
What? We need to use comments to track specific sections of text. Do you mean in general or just here? It appears to me the other way around, translations are breaking the doc. We cannot do our job and fix errors that crop up, update text from our vale reports, use comments etc...all because of translation then this becomes rather chaotic. Do you have another solution to not using comments? Creating issues is incorrect because then the comment is separate from the text itself and it will become a nightmare for maintaining such things.